### ISSUE TITLE
ok, so I forgot to use long double floats because non-existent brain cells

### ISSUE THREAD

**Issue by rickdegraaf1** [rickdegraaf1/ultracalculator#1]
I will update it when I have time (probably in a month)

**Comment by SemPruijs** [comment]
Cool, is this your first issue on GitHub?

**Comment by 277r** [comment]
> Cool, is this your first issue on GitHub?

Yeah, but I'm too lazy to work on this since I'm already working on something big

**Comment by SemPruijs** [comment]
Cool, on the youtube thing?

**Comment by 277r** [comment]
No, something else. But if we are going to continue this conversation like this you Might aswell add me on Discord
(this message will be deleted) RS#9699

**Comment by SemPruijs** [comment]
<img width="1340" alt="Screen Shot 2020-08-05 at 1 22 33 PM" src="https://user-images.githubusercontent.com/19208988/89407270-c7d77b80-d71e-11ea-9a54-772a5997ba7c.png">
You're link does not work

**Comment by 277r** [comment]
Hmmm Weird, I think I allowed people to
![image](https://user-images.githubusercontent.com/38425500/89407457-256bc800-d71f-11ea-8a66-5fd45c1610fa.png)
Do you know what setting it might be?
maybe adding as friend fixes it, since I got that allowed

**Comment by SemPruijs** [comment]
mmmm, lets try this https://discord.gg/HAAGFX
